docker-compose.yml

```diff
@@ -103,6 +103,11 @@ services:
       - "traefik.http.routers.to-scheduler.middlewares=scheduler-stripprefix@docker"
       - "traefik.http.middlewares.scheduler-stripprefix.stripprefix.prefixes=/api/scheduler"
 
+  scheduler-db:
```

Comment:
Why did you add persistence to the scheduler? I think you should remove the scheduler's persistence since we will keep the state in memory. Or is there something I am missing here?

Reply:
I added it by mistake. I will remove it with the next commit